frontend/modules/issues/issues.redux.ts

```diff
@@ -147,10 +147,17 @@ export const saveIssueSuccess = (state = INITIAL_STATE, { issue }) => {
 		issuesMap[state.componentState.activeIssue].position = oldPosition;
 	}
 
+	if (resetComponentState) {
```

Comment:
Can we reduce duplication here? Do something like this?

```suggestion
	const newComponentState = { ...state.componentState };
	if (resetComponentState) {
	      newComponentState.newIssue = {};
	}
	return {
			...state,
			issuesMap,
			componentState: newComponentState
		};
```

Reply:
Of course, it looks better now, thanks for spotting this.